packages/experimental/AppearanceAdditions/src/appearanceAdditions.ts

```diff
@@ -0,0 +1,7 @@
+import { IAppearanceAdditions } from './NativeAppearanceAdditions.types';
+
+// Default values for non-iOS clients.
+export const appearanceAdditions = {
+  horizontalSizeClass: 'regular',
+  userInterfaceLevel: 'base',
+} as IAppearanceAdditions;
```

Comment:
Nit: We don't like the `I` prefix anymore in FURN, and avoid it for new types.

Reply:
Aw, but I like the `I` prefix :( okay removing it